basic-application/basic-application-webapp/src/main/java/fr/openwide/core/basicapp/web/application/history/column/HistoryLogDetailColumn.java

```diff
@@ -0,0 +1,79 @@
+package fr.openwide.core.basicapp.web.application.history.column;
```

Comment:
I wonder if we should add this class (and those on which it depends) to the core... Its purpose is to provide a configurable structure, after all, and I think we will be able to override much of this in applications:
 * Particular components for particular fields thanks to the factory system
 * Different UI style thanks to CSS override (the HTML code is quite simple)

What do you think?